wip: initial (#143)
FlatFilers/flatfile-plugins: plugins/dedupe/README.md

# @flatfile/plugin-dedupe

> asdf

Reviewer: Can you write a brief description of the package's purpose?

> Dedupe records in a sheet via a sheet level custom action.

Reviewer: ```suggestion
Dedupe records in a sheet via a sheet level custom transformation.
```

`npm i @flatfile/plugin-dedupe`

## Get Started

> Follow [this guide](https://flatfile.com/docs/plugins/asdf/asdf) to learn how to use the plugin.

Reviewer: ```suggestion
Follow [this guide](https://flatfile.com/docs/plugins/automations/dedupe) to learn how to use the plugin.
```

> Follow [this guide](https://flatfile.com/docs/plugins/automations/dedupe) to learn how to use the plugin.

Reviewer: ```suggestion
Follow [this guide](https://flatfile.com/docs/plugins/transform/dedupe) to learn how to use the plugin.
```
